boot/zephyr/boards/nrf54l15pdk_nrf54l15_cpuapp_ext_flash.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_MULTITHREADING=y
+CONFIG_SPI=y
+CONFIG_SPI_NOR=y
+CONFIG_FLASH=y
+CONFIG_PM_PARTITION_SIZE_MCUBOOT=0x14000
```

Comment:
This implies usage of Partition Manager, is that correct?

Reply:
yes